README.md

```diff
@@ -2,9 +2,10 @@
 
 [![Build Status](https://travis-ci.org/flix-tech/danger-code_style_validation.svg?branch=master)](https://travis-ci.org/flix-tech/danger-code_style_validation)
 
-This plugin looks for code style violations for added lines and suggests patches.
-
-It uses 'clang-format' and only checks `.h`, `.m` and `.mm` files
+This plugin uses 'clang-format' to look for code style violations in added
+lines on the current MR / PR, and offers inline patches.
+By default only Objective-C files, with extensions `.h`, `.m`, `.mm` and
+`.C`, are checked.
```

Comment:
According to [Wikipedia](https://en.wikipedia.org/wiki/Objective-C) Objective-C files usually have those extensions.